<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BomberMan </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>141</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxStat">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Statistics</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>130</y>
          <width>141</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBoxInfosGame">
           <property name="title">
            <string/>
           </property>
           <widget class="QLabel" name="labelLife">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>31</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Life</string>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLifeOne">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>10</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/icon_coeur.png</normaloff>../../../../Pictures/icon_coeur.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLifeFour">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>84</x>
              <y>10</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/icon_coeur.png</normaloff>../../../../Pictures/icon_coeur.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLifeThree">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>66</x>
              <y>10</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/icon_coeur.png</normaloff>../../../../Pictures/icon_coeur.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLifeTwo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>48</x>
              <y>10</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/icon_coeur.png</normaloff>../../../../Pictures/icon_coeur.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonLifeFive">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>102</x>
              <y>10</y>
              <width>21</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/icon_coeur.png</normaloff>../../../../Pictures/icon_coeur.png</iconset>
            </property>
           </widget>
           <widget class="QPushButton" name="pushButtonScore">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>30</y>
              <width>21</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../Pictures/score.png</normaloff>../../../../Pictures/score.png</iconset>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEditScore">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>51</x>
              <y>31</y>
              <width>41</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="labelScore">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>35</y>
              <width>46</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Score</string>
            </property>
           </widget>
           <widget class="QLabel" name="labelBonus">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>60</y>
              <width>46</width>
              <height>13</height>
             </rect>
            </property>
            <property name="text">
             <string>Bonus </string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>141</width>
          <height>91</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBoxInfosPlayer">
           <property name="title">
            <string/>
           </property>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>71</width>
              <height>79</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="labelTimer">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelLevel">
               <property name="text">
                <string>Level</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelUserName">
               <property name="text">
                <string>Player name</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>71</x>
              <y>0</y>
              <width>71</width>
              <height>80</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTimeEdit" name="TimeEditTimer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="frame">
                <bool>true</bool>
               </property>
               <property name="timeSpec">
                <enum>Qt::LocalTime</enum>
               </property>
               <property name="time">
                <time>
                 <hour>0</hour>
                 <minute>0</minute>
                 <second>0</second>
                </time>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditLTimer"/>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditPlayerName"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Jeu</string>
    </property>
    <widget class="QMenu" name="menuNouvelle_Partie">
     <property name="title">
      <string>Nouvelle Partie</string>
     </property>
     <widget class="QMenu" name="menuPartie_solo">
      <property name="title">
       <string>Partie solo</string>
      </property>
      <addaction name="actionDemarrerPartieSolo"/>
      <addaction name="actionSauvegarderPartieSolo"/>
      <addaction name="actionChargerPartieSolo"/>
     </widget>
     <widget class="QMenu" name="menuPartie_multi_joueur">
      <property name="title">
       <string>Partie multi-joueur</string>
      </property>
      <addaction name="actionDemarrerPartieMulti"/>
      <addaction name="actionChargerPartieMulti"/>
     </widget>
     <addaction name="menuPartie_solo"/>
     <addaction name="menuPartie_multi_joueur"/>
    </widget>
    <widget class="QMenu" name="menuOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <addaction name="actionCommandes_de_jeu"/>
     <addaction name="actionAffichage"/>
     <addaction name="actionAudio"/>
    </widget>
    <widget class="QMenu" name="menuProfil">
     <property name="title">
      <string>Profil</string>
     </property>
     <addaction name="actionInformations"/>
     <addaction name="actionLeaderboard"/>
     <addaction name="actionAffichageStatistics"/>
     <addaction name="actionModifier_mot_de_passe"/>
    </widget>
    <addaction name="menuNouvelle_Partie"/>
    <addaction name="menuOptions"/>
    <addaction name="menuProfil"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAide"/>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDemarrerPartieSolo">
   <property name="text">
    <string>Démarrer partie</string>
   </property>
  </action>
  <action name="actionSauvegarderPartieSolo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sauvegarder partie</string>
   </property>
  </action>
  <action name="actionDemarrerPartieMulti">
   <property name="text">
    <string>Démarrer partie</string>
   </property>
  </action>
  <action name="actionChargerPartieMulti">
   <property name="text">
    <string>Charger partie</string>
   </property>
  </action>
  <action name="actionChargerPartieSolo">
   <property name="text">
    <string>Charger partie</string>
   </property>
  </action>
  <action name="actionCommandes_de_jeu">
   <property name="text">
    <string>Commandes de jeu</string>
   </property>
  </action>
  <action name="actionAffichage">
   <property name="text">
    <string>Affichage</string>
   </property>
  </action>
  <action name="actionAudio">
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionInformations">
   <property name="text">
    <string>Informations</string>
   </property>
  </action>
  <action name="actionLeaderboard">
   <property name="text">
    <string>Leaderboard</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAide">
   <property name="text">
    <string>Aide</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionAffichageStatistics">
   <property name="text">
    <string>Satistics</string>
   </property>
  </action>
  <action name="actionModifier_mot_de_passe">
   <property name="text">
    <string>Modifier mot de passe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
